app.rb

```diff
@@ -0,0 +1,74 @@
+require_relative './lib/item_repository'
+require_relative './lib/order_repository'
+require_relative './lib/database_connection'
+
+class Application
+    database_name = 'store_manager_test'
+
+  def initialize(database_name, io, item_repository, order_repository)
+    DatabaseConnection.connect(database_name)
+    @io = io
+    @item_repository = item_repository
+    @order_repository = order_repository
+  end
+
+  def run
+    @io.puts "Welcome to the shop management program! \n\n"
+    @io.puts "What do you want to do?
+      1 = list all shop items
+      2 = create a new item
+      3 = list all orders
+      4 = create a new order"
+     choice = @io.gets.chomp
+    case choice
+    when '1'
+      @item_repository.all.each do |item|
+        @io.puts "##{item.id} #{item.product} - Unit price: #{item.price} - Quantity: #{item.quantity}"
+        run
+      end
```

Comment:
refactor this bad boy, when '1' then call_a_method, when '2' then call_another_method